bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,162 @@
+  solvers/expose-solvers.hpp
+  solvers/solver-HQP-eiquadprog.hpp
+  solvers/HQPData.hpp
+  solvers/HQPOutput.hpp
+  formulations/formulation.hpp
+  formulations/expose-formulations.hpp
+  )
+
+SET(${PROJECT_NAME}_PYTHON_SOURCES
+  module.cpp
+  robots/robot-wrapper.cpp  
+  constraint/constraint-bound.cpp
+  constraint/constraint-equality.cpp
+  constraint/constraint-inequality.cpp
+  contacts/contact.cpp
+  trajectories/trajectory-euclidian.cpp
+  trajectories/trajectory-se3.cpp
+  trajectories/trajectory-base.cpp 
+  tasks/task-com-equality.cpp
+  tasks/task-se3-equality.cpp
+  tasks/task-joint-posture.cpp
+  solvers/solver-HQP-eiquadprog.cpp 
+  solvers/HQPData.cpp
+  solvers/HQPOutput.cpp
+  formulations/formulation.cpp
+  )
+
+
+LIST(APPEND HEADERS ${${PROJECT_NAME}_PYTHON_HEADERS})
+LIST(REMOVE_DUPLICATES HEADERS)
+
+# Headers of the python binding
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/robots")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/constraint")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/contacts")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/trajectories")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/tasks")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/solvers")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/formulations")
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/tsid/bindings/python/utils")
+
+SYMLINK_AND_INSTALL_HEADERS("${${PROJECT_NAME}_PYTHON_HEADERS}" "bindings/python")
+
+# --- COMPILE WRAPPER
+MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/bindings/python/${PROJECT_NAME}")
+SET(${PYWRAP}_SOURCES ${${PROJECT_NAME}_PYTHON_SOURCES})
+SET(${PYWRAP}_HEADERS ${${PROJECT_NAME}_PYTHON_HEADERS})
+
+ADD_LIBRARY(${PYWRAP} SHARED ${${PYWRAP}_SOURCES} ${${PYWRAP}_HEADERS})
+ADD_DEPENDENCIES(python ${PYWRAP})
+
+IF(BUILD_WITH_COMMIT_VERSION)
+  TAG_LIBRARY_VERSION(${PYWRAP})
+ENDIF(BUILD_WITH_COMMIT_VERSION)
+ADD_HEADER_GROUP(${PYWRAP}_HEADERS)
+ADD_SOURCE_GROUP(${PYWRAP}_SOURCES)
+PKG_CONFIG_USE_DEPENDENCY(${PYWRAP} eigenpy)
+
+TARGET_LINK_LIBRARIES(${PYWRAP} ${PROJECT_NAME})
+TARGET_LINK_BOOST_PYTHON(${PYWRAP})
+
+IF(APPLE)
+  # We need to change the extension for python bindings
+  SET_TARGET_PROPERTIES(${PYWRAP} PROPERTIES SUFFIX ".so")
+ENDIF(APPLE)
+
+SET_TARGET_PROPERTIES(${PYWRAP} PROPERTIES
+  LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bindings/python/${PROJECT_NAME}")
+
+INSTALL(TARGETS ${PYWRAP} DESTINATION ${${PYWRAP}_INSTALL_DIR})
+
+# --- INSTALL SCRIPTS 
+SET(PYTHON_FILES
+ __init__.py
+  )
+
+FOREACH(python ${PYTHON_FILES})
+  INSTALL(FILES
+    "${${PROJECT_NAME}_SOURCE_DIR}/bindings/python/${python}"
+    DESTINATION ${${PYWRAP}_INSTALL_DIR})
+ENDFOREACH(python)
+
+# --- PACKAGING --- #
+
+# Format string
```

Comment:
You have to suppress all these packaging information.
They are mandatory for Pinocchio, not here.